postmortems/000 - Postmortem Template.md

```diff
@@ -0,0 +1,37 @@
+# 000 - Postmortem for [descriptive name here]
+
+## Level: [Internal, Outage, Corruption]
+
+## Author(s)
+
+- Some Body <oncetoldme@iasql.com>
+
+## Summary
+
+This is a postmortem template for outages. In this section you should put a short summary of what happened and how it was resolved. Only one or two paragraphs at most.
+
+Also, to explain the levels. An "Internal" outage is when our internal dev processes are broken and negatively impact the team. An "Outage" level is when service is interrupted for users, but zero impact on customer cloud accounts occurred. A "Corruption" level we hope to never see, which would imply accidental destruction of cloud resources in a customer account caused by us.
```

Comment:
i can think of other type, such as a security breach. Is not corruption, but confidential data exposed to an attacker

Reply:
This is a good idea, let me add it.